documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc

```diff
@@ -48,28 +49,33 @@ public class ComponentContainer extends PolymerTemplate<TemplateModel> {
 }
 
 ----
-
-Without the *slot* tag in the template the added label and button would not be visible to the user even though it can be located in the DOM.
-As we can see multiple components will show up in the slot when added to the template element that has an open `<slot></slot>`
+* Without the `<slot>` tag in the template, the label and button would not be visible to the user, even though they can be located in the DOM.
+* You can add multiple components and they will display in the slot when added to the template element that has an open `<slot></slot>`.
 
 [NOTE]
-Any element added to the light dom to be shown inside a `<slot>` can be removed from the dom and it will work as expected and disappear from the main element.
+You can remove any element from a `<slot>` and it will work as expected, and no longer be displayed in the main element.
```

Comment:
Maybe reword to: 

You can remove any element from a `<slot>` - that will work as expected, causing the element no longer to be displayed in the main element.

Reply:
What do you think about something along these lines:
```
You can remove any element from a `<slot>` and it will work as expected, the element will no longer be displayed as part of the main element.
```

Reply:
Sounds good!